_posts/2022-03-13-software.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Data"
+date:   2022-03-23 
+author: Nandi Moksnes
+category: Open Science
+tags: software release
+---
+
+# Software release
+
+Many of the CCG research teams work with version control such as Git and Github. However, the scientific code could also be released under a e.g. Python package if the code is in Python. One such channel is [Python Package Index (PyPi)](https://pypi.org/) where the packages can be distributed under different versions.
+  
+![image](https://user-images.githubusercontent.com/30128518/159683921-d979dd9c-0db3-4f99-b581-c62870562cab.png)
+
+A software release covers all of the principles for open science and a brief decsription can be found below and longer decription can be found in the respective principle.
+  
+### [Workflow](https://climatecompatiblegrowth.github.io/guidelines/open%20science/2022/03/18/workflow.html)
+When developing a workflow for a software release it is important to develop clear objects and steps with tests so that bugs and issues can be identified and reported. This is to assure that the workflow is working as intended at the release and that bugs or issues can be fixed for next release. More advanced software development has tests inplace which are checked for each change in code to ensure consistency. One such integrated tool on Github is [Travis CI](https://docs.travis-ci.com/user/tutorial/) which builds the code and tests it with user defined tests. Travis CI has a payment plan, but for open source accounts it is free under travis-ci.com.
```

Comment:
I think here it is more important to zoom out and provide the context to the reader. Why would you want to release a workflow? Who would it help? How does it support Open Science and reproducible research?

Reply:
I added a sentence on Open science and resuability. Let me know if that is more what you had in mind.